jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/model/EventMatcherType.java

```diff
@@ -40,14 +40,7 @@
      * We need to add multiple events to the issue transitioned cause Jira classic workflow send specific event types
```

Comment:
Does this comment make sense now?

Reply:
No, it doesn't! I'll fix this comment in the next PR.